docker-entrypoint.sh

```diff
@@ -23,7 +23,9 @@ fi
 if [ -d "/usr/local/tomcat/webapps/ors" ]; then
 	cp -f /ors-conf/app.config $tomcat_appconfig
 else
-	cp -f $source_appconfig /ors-conf/app.config
```

Comment:
This copies the default app.config and ignores the users app.config

Reply:
OK, I think I see now some more of the logic after looking into the Dockerfile and docker-compose file. So, for the user to define their own app.config, they wouldn't place it into the folder on the host mapped to the internal ors-conf folder (or if they did, wouldn't name it as app.config) but need to tell Docker the path of the file using the APP_CONFIG arg (pointing to a file on the host within the build context). The file defined by this then overwrites the app.config.sample file inside the container, which is then renamed to app.config so that ors can use it. Then the paths within it modified to point to the correct folders within the container (this all happens in the Dockerfile). What the docker-entrypoint stuff does is to copy that file to  `/ors-conf/app.config` so that the user can see within the host (as long as it is mapped as a volume) what config is being used and make modifications to it. Then when the container is restarted this config file gets copied to overwrite the one in tomcat which is where openrouteservice is actually running from.

So yes, I think what your code is doing is correct in that it overwrites the custom file if it doesnt exist, but to get the custom config to be used when ors is first built, it needs to be passed as the `APP_CONFIG` arg in docker and not placed in the ors-conf folder mapped to the host.

Reply:
Yes, that seems to be the current logic. However, that does not match any use case I can think of, as the further lines in the dockerfile scrutinize the custom app.config. Also this makes the image published on docker hub quite useless.